fabulaws/ubuntu/instances/base.py

```diff
@@ -76,6 +77,32 @@ def _encrypt_device(self, device, passwd=None):
         return device
 
     @uses_fabric
+    def _mount_and_persist(self, device, mount_point, fs_type=None, opts=None):
+        mount_cmd = ' '.join([
+            'mount',
+            '-o {}'.format(opts) if opts else '',
+            '-t {}'.format(fs_type) if fs_type else '',
+            device,
+            mount_point,
+        ])
+        fstab_entry = ' '.join([
```

Comment:
I liked having it close to `mount_cmd` so you could easily see how they differ...